other/refresh-env-var-in-pod/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -25,21 +25,23 @@ spec:
         file: deployment.yaml
   - name: step-03
     try:
+    - sleep:
+        duration: 5s
     - script:
         content: |
           kubectl get pod -n refresh-env-var-ns --no-headers | awk '{print $1}' > deploy-names.txt
   - name: step-04
     try:
+    - sleep:
+        duration: 5s
     - apply:
         file: chainsaw-step-04-apply-1.yaml
     - apply:
         file: chainsaw-step-04-apply-2.yaml
   - name: step-05
     try:
-    - command:
-        args:
-        - "5"
-        entrypoint: sleep
+    - sleep:
```

Comment:
do we need to sleep here ?

Reply:
Strictly speaking? Maybe not. But it may help. And it's short.